src/VisualStudio/CSharp/Impl/VSPackage.resx

```diff
@@ -363,4 +368,4 @@ Show items from unimported namespaces (experimental);</value>
   <data name="An_empty_CSharp_script_file" xml:space="preserve">
     <value>An empty C# script file.</value>
   </data>
-</root>
\ No newline at end of file
+</root>
```

Comment:
Added by editing online. If it's not a problem, maybe better to keep to allow for easier online editing going forward.